### PR TITLE
Feat(Task-8): As a User, I want a Custom HTML5 Video Player

### PR DESCRIPTION
**Pull request by SamuFigueirido**

### CODE REVIEW

File: Exercise11/scripts.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    }
+}
+
+function updateButton() {
+    const icon = this.paused ? '►' : '❚❚';
+    toggle.textContent = icon;
+}
+
+function skip() {
+    console.log(this.dataset.skip);
```
Comment (eloydotco) [inline Exercise11/scripts.js]: Please, remove this log

**Review by eloydotco — APPROVED** [review]

File: Exercise11/scripts.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+video.addEventListener('pause', updateButton);
+toggle.addEventListener('click', togglePlay);
+
+skipButtons.forEach(button => button.addEventListener('click', skip));
+
+ranges.forEach(range => range.addEventListener('change', handleRangeUpdate));
+
+video.addEventListener('timeupdate', handleProgress);
+
+let mousedown = false;
```
Comment (gbtec-ext-jorgevillar) [inline Exercise11/scripts.js]: I would put this variable at the beginning where you define the consts

File: Exercise11/style.css [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+/* This css is only applied when fullscreen is active. */
+.player:fullscreen {
+  max-width: none;
+  width: 100%;
+}
+
+.player:-webkit-full-screen {
+  max-width: none;
+  width: 100%;
+}
```
Comment (gbtec-ext-jorgevillar) [inline Exercise11/style.css:35]: This class has the same styles as the one above, please, join them

**Review by gbtec-ext-jorgevillar — CHANGES_REQUESTED** [review]

File: Exercise11/style.css [hunk 4]
```diff
@@ -28,12 +28,7 @@ body {
 }
 
 /* This css is only applied when fullscreen is active. */
-.player:fullscreen {
-  max-width: none;
-  width: 100%;
-}
-
-.player:-webkit-full-screen {
+.player:-webkit-full-screen, .player:fullscreen {
```
Comment (gbtec-ext-jorgevillar) [inline Exercise11/style.css]: This is only a personal preference:
I prefer to put each selector in different line:
.player:fullscreen, 
.player:-webkit-full-screen {
If you have a lot of the or they are longer I think it's more readable this way

**Review by gbtec-ext-jorgevillar — APPROVED** [review]

**Pull request merged**